from urllib3.exceptions import HTTPError
import urllib3


dir(urllib3)